### PR TITLE
rename CLISP patch + add upstream ref + minor style cleanups in easyconfig

### PR DESCRIPTION
**Pull request by boegel**
follow-up for #4926

cc @JackPerdue & @verdurin (since you merged it)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1e444854c21494f483001d7f50fc73f5 for a full test report.

File: easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -14,12 +14,11 @@ description = """
 
 toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
 
-source_urls = ['ftp://ftp.gnu.org/pub/gnu/%(namelower)s/release/%(version)s']
+source_urls = [GNU_SOURCE + '/release/%(version)s']
 sources = [SOURCELOWER_TAR_BZ2]
 checksums = ['8132ff353afaa70e6b19367a25ae3d5a43627279c25647c220641fed00f8e890']
```
Comment (vanzod) [inline easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb]: No patch checksum?

Comment (boegel) [inline easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-6.4.0.eb]: OK, I'll add it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0eda5ec83c30f30fe320848bb7c4ec46 for a full test report.

**Comment by JackPerdue** [comment]
Looks OK here.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
node2573.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/635111949d48196fca27d8dcd8ae23d3 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**